### PR TITLE
Improve Cache Logging

### PR DESCRIPTION
**Pull request by ghost**
See the comment associated with the first commit.

### CODE REVIEW

File: lib/components/rails/caching.rb [hunk 1]
```diff
@@ -34,6 +34,18 @@ def cache_fragment_name
       def virtual_path
         "#{component_path}/#{action_name}"
       end
+
+      def instrument_payload(key)
```
Comment (benhutton) [inline lib/components/rails/caching.rb]: @joeosburn-desiringgod where do you think we are triggering code that uses this?

**Comment by benhutton** [comment]
@joeosburn-desiringgod can't you point a local branch of www at this branch here, enable caching, and test it? No need for review apps - you should be able to do it in development.

File: lib/components/rails/caching.rb (hunk 1 above)
Comment (joeosburn-desiringgod) [reply lib/components/rails/caching.rb]: @benhutton https://github.com/rails/rails/blob/b5db73076914e7103466bd76bec785cbcfe88875/actionpack/lib/abstract_controller/caching/fragments.rb#L161-L163

**Comment by joeosburn-desiringgod** [comment]
@benhutton Check out the latest commit here.

**Review by benhutton — APPROVED** [review]

**Pull request merged**